src/core/ext/transport/chttp2/client/secure/secure_channel_create.c

```diff
@@ -97,15 +98,15 @@ grpc_channel *grpc_secure_channel_create(grpc_channel_credentials *creds,
   GRPC_API_TRACE(
       "grpc_secure_channel_create(creds=%p, target=%s, args=%p, "
       "reserved=%p)",
-      4, (creds, target, args, reserved));
+      4, ((void *)creds, target, (void *)args, (void *)reserved));
   GPR_ASSERT(reserved == NULL);
   // Make sure security connector does not already exist in args.
```

Comment:
Creating a new security connector for each subchannel creation may not be a good idea from the performance standpoint. Regular non-LB connections will start paying the price of this creation.

Something else we lose when we move the all security connector creations to the subchannel level is the ability to report errors via a lame client channel, as currently done in `grpc_secure_channel_create`